gammapy/cube/models.py

```diff
@@ -340,3 +341,48 @@ def evaluate(self, lon, lat, energy):
     def copy(self):
         """A shallow copy"""
         return copy.copy(self)
+
+
+class BackgroundModel(Model):
+    """Background model
+
+    Parameters
+    ----------
+    background : `~gammapy.maps.Map`
+        Background model map
+    norm : float
+        Background normalisation
+    tilt : float
+        Additional tilt in the spectrum
+    reference : `Quantity`
+        Reference energy of the tilt.
+    """
+    def __init__(self, background, norm=1, tilt=0, reference="1 TeV"):
+
+        axis = background.geom.get_axis_by_name("energy")
+        if axis.node_type != "edges":
+            raise ValueError('Need an integrated map, energy axis node_type="edges"')
+
+        self.map = background
+        self.parameters = Parameters([
+            Parameter("norm", norm, unit=""),
+            Parameter("tilt", tilt, unit="", frozen=True),
+            Parameter("reference", reference, frozen=True),
+        ])
+
+
```

Comment:
I think it might be a good option to cache the energy array using the `@lazyproperty` decorator from `astropy.utils`. Not really critical, but why not, if it comes for free...

Reply:
Done!